### DISCUSSION TITLE
Supported platforms

### DISCUSSION THREAD

**Discussion by CrayolaEater** [h2zero/NimBLE-Arduino#666] [Q&A]
What are the supported platforms of this library ? Does it support Android, IOS, Win, Linux out of the box?

**Comment by h2zero** [comment]
This only supports esp32 and nrf5 platforms at this time.

**Reply by CrayolaEater** [reply]
is there an alternative for all those platforms?

**Reply by h2zero** [reply]
I don't know of anything that works on all of those, sorry.
